helm/charts/6/6.0.0-GA/skywalking/values.yaml

```diff
@@ -105,3 +105,5 @@ ui:
     requests:
      # cpu: 100m
      memory: 1Gi
+
+TZ: Asia/Shanghai
```

Comment:
Please don't set TZ to China. We don't know the user timezone. UTC 0 as default is better.

Reply:
You have thought very carefully. @wu-sheng